src/KM_MapEditor.pas

```diff
@@ -215,12 +218,14 @@ function TKMMapEditor.HumanCount: Integer;
 end;
 
 
-procedure TKMMapEditor.UpdateField(aInc: Integer; aCheckPrevCell: Boolean);
+//aStageIncrement - stage increment, can be negative

```

Comment:
This is a bad sort of comment. It says exactly what argument name says - that this is a "stage" and "increment" and it has signed type.
Could you make it more "helpful" instead?